src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -143,7 +143,8 @@ internal abstract partial class AbstractFindUsagesService : IFindUsagesService
                 project.Solution,
                 progressAdapter,
                 documents: null,
-                cancellationToken: cancellationToken).ConfigureAwait(false);
+                new SymbolFinderOptions(cascade: false, includeImplicitAccessorUsages: true),
```

Comment:
is this just for testing purposes?  because it seems not-good that you globally turned off cascade for FindSymbolReferencesAsync. right?   Is hte intent that we would properly check if you're on the accessor declaration, and if so set these options?

Reply:
Yes. I'll be able to come back through and update theses things late tonight.